README.md

```diff
@@ -38,20 +38,30 @@ If you're using the Lando-based environvment and it's already running, you can r
 lando test
 ```
 
-If you don't have the Lando-based environment running (e.g. in a CI context), we have a script that runs unit tests in a self-contained Docker environment.  To run these tests, execute the following from the project root:
+If you don't have the Lando-based environment running (e.g. in a CI context), we have a script that runs unit tests in a self-contained Docker environment. To run these tests, execute the following from the project root:
 
 ```bash
-./bin/phpunit-docker.sh [wp-version]
+./bin/phpunit-docker.sh
 ```
 
 You can either pass a version number to test against a specific version, or leave it blank to test against the latest version.
 
+```bash
+./bin/phpunit-docker.sh --version 5.4.4
+```
+
 You can also pass the path to a specific test as well as extra PHPUnit arguments:
```

Comment:
I don't think this line is true anymore because we only pass `--stop-on-failure` and the path to test with the new changes?

TBF, I'm not really sure whether arguments passthrough worked though.

Reply:
Actually, that command _does_ work. We no longer pass the version as a parameter to the bootstrap, but that doesn't matter. Notice that the version is defined by what's mounted as a volume, which still uses the `$WP_VERSION` variable.